contracts/StrategyAuraStaking.sol

```diff
@@ -293,6 +293,9 @@ contract StrategyAuraStaking is BaseStrategy {
         view
         returns (uint256 amount)
     {
+        uint256 mintAmount = _balAmount
+            .mul(BOOSTER.getRewardMultipliers(address(baseRewardPool)))
+            .div(BOOSTER.REWARD_MULTIPLIER_DENOMINATOR());
```

Comment:
I would prefer to move this out of `getMintableAuraRewards` and call the function with the adjusted bal value

```solidity
uint256 balAmountAdjusted = _balAmount.mul(BOOSTER.getRewardMultipliers(address(baseRewardPool))).div(REWARD_MULTIPLIER_DENOMINATOR;
getMintableAuraRewards(balAmountAdjusted);
```

Reply:
Good call, pushed